src/Abp/Application/Features/IFeatureValueStore.cs

```diff
@@ -3,7 +3,7 @@
 namespace Abp.Application.Features
 {
     /// <summary>
-    /// Defines a store to get feature values.
+    /// Defines a store to get a feature's values.
```

Comment:
Should `values` be `value` since you added `a` (A feature has a single value)?

Reply:
Sure!